hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -95,7 +99,10 @@ public CallOutcome call() {
         assertEthTxDataValidIfApplicable();
 
         // Try to translate the HAPI operation to a Hedera EVM transaction, throw HandleException on failure
```

Comment:
This (existing) comment might stand improvement.  Because now there's success, `isException`, and failure.